src/components/DeliverablesTable/index.tsx

```diff
@@ -50,10 +49,15 @@ const DeliverablesTable = ({
   const dispatch = useAppDispatch();
   const { activeLocale } = useLocalization();
   const { isAllowed } = useUser();
-  const { selectedOrganization } = useOrganization();
-  const { availableProjects: projects, getProjectName } = useProjects();
+  const { acceleratorOrganizations, selectedOrganization } = useOrganization();
   const { selectedParticipant } = useParticipants(participantId);
 
+  const acceleratorProjects = useMemo(() => {
+    return acceleratorOrganizations.reduce((acc: AcceleratorProject[], org: AcceleratorOrganization) => {
```

Comment:
this can be achieved with a flatMap  `acceleratorOrganizations.flatMap(org => org.projects)`, will return a flat list of all the project ids